R/geom-rug.r

```diff
@@ -66,6 +66,10 @@ GeomRug <- ggproto("GeomRug", Geom,
     rugs <- list()
     data <- coord$transform(data, panel_params)
 
+    if (inherits(coord, 'CoordFlip')) {
```

Comment:
I wish we wouldn't test for capabilities by testing for inheritance, but this is how it's done for `CoordFlip` all over the ggplot2 code base, so I guess it's fine.

Reply:
The first instinct was to use `is()`, but it was not used anywhere in the coord base.

Reply:
What I meant was other coord features are queried by asking the coord, e.g. `coord$is_free()`. So it would probably make sense to introduce `coord$is_flip()`. But that's for a different PR.